src/test/resources/localstack/setup-aws.sh

```diff
@@ -7,7 +7,10 @@ export AWS_DEFAULT_REGION=eu-west-2
 export PAGER=
 
 # Create the bucket
-aws s3 --endpoint-url=http://localhost:4566 --region eu-west-2 ls s3://local-644707540a8083b7b15a77f51641f632 || aws --endpoint-url=http://localhost:4566 --region=eu-west-2 s3 mb s3://local-644707540a8083b7b15a77f51641f632
+aws s3 --endpoint-url=http://localhost:4566 --region eu-west-2 ls s3://cpg-s3-bucket  || aws --endpoint-url=http://localhost:4566 --region eu-west-2 s3 mb s3://cpg-s3-bucket
+aws s3 --endpoint-url=http://localhost:4566 --region eu-west-2 ls s3://s3-large-cases-bucket  || aws --endpoint-url=http://localhost:4566 --region eu-west-2 s3 mb s3://s3-large-cases-bucket
 
-echo "S3 Configured with cpg-bucket"
+aws --endpoint-url=http://localhost:4566 sns create-topic --name court-cases-topic.fifo --attributes '{"FifoTopic":"true", "ContentBasedDeduplication":"true"}'
+aws --endpoint-url=http://localhost:4566 sns subscribe --topic-arn "arn:aws:sns:eu-west-2:000000000000:court-cases-topic.fifo" --protocol "sqs" --notification-endpoint "arn:aws:sns:eu-west-2:000000000000:court_cases_queue.fifo"
```

Comment:
Creating the SNS topic and subscription in here is not necessary - see https://github.com/ministryofjustice/hmpps-spring-boot-sqs?tab=readme-ov-file#hmpps-queue-properties

> create LocalStack queues and topics for testing against, and subscribe queues to topics where configured